CMakeLists.txt

```diff
@@ -109,4 +109,7 @@ install(FILES "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}-config.cmake"
 include(CTest)
 if (BUILD_TESTING)
     add_subdirectory(tests)
+    if (NOT CMAKE_CROSSCOMPILING )
+        add_subdirectory(bin/elastipubsub)
```

Comment:
Bugfix looks good, but don't have time to review the sample app

can we leave that out for now? or would it slow you down testing further stuff?

Reply:
Let's leave in for now.